javaparser-core/src/main/java/com/github/javaparser/ast/visitor/SimpleVoidVisitor.java

```diff
@@ -0,0 +1,337 @@
+package com.github.javaparser.ast.visitor;
```

Comment:
This is used in https://github.com/Bohdankm22/checker-framework/blob/ec72c7ca8a3ccc2178df46a5478d4e7b867aa36f/framework/src/org/checkerframework/framework/stub/StubUtil.java#L9